deploy/kubernetes/kubernetes.adoc

```diff
@@ -0,0 +1,87 @@
+:github-address: https://github.com/hazelcast-guides/hazelcast-embedded-springboot
+
+= Deploy to Kubernetes
+
+This guide will help you to deploy your application to Kubernetes environment.
+
+== Prerequisites
+
+- ~15 minutes
+- https://docs.docker.com/install/[Docker]
+- A https://kubernetes.io/[Kubernetes] environment. We will use local `minikube` environment as k8s for demonstration.
+
+== Updating Hazelcast Configuration
+
+Hazelcast has a Kubernetes Discovery Plugin which helps to build clusters in a Kubernetes environment. To enable Kubernetes discovery, update your `hazelcast.yaml` configuration as below:
+
+[source, yaml]
+----
+hazelcast:
+  # other Hazelcast configurations
+  network:
+    join:
+      multicast:
+        enabled: false
+      kubernetes:
+        enabled: true
+----
+
+== Containerizing the Application
+
+Firstly, please assure that Docker is running on your environment.
+
+In order to create the Docker image of the application, we will use the https://github.com/GoogleContainerTools/jib[Jib] tool, and its related Maven plugin. It allows to build containers from Java applications without a Docker file or even changing the pom.xml file. To build the image, run the command below:
+
+----
+mvn clean compile com.google.cloud.tools:jib-maven-plugin:2.1.0:dockerBuild
```

Comment:
I would probably show the whole process with `Dockerfile`, JIB adoption is not so high and most people still use Dockerfiles, even for Spring Boot apps.